<launch>
    <!-- 加载传感器配置（使用 mag_sensor_node 的配置）-->
    <rosparam command="load" file="$(find mag_sensor_node)/config/sensor_config.yaml" />

    <!-- 加载可视化配置 -->
    <rosparam command="load" file="$(find mag_pose_visualization)/config/visualization_config.yaml" />

    <!-- 启动可视化节点 -->
    <node pkg="mag_pose_visualization" type="magnetic_field_visualizer_node" name="magnetic_field_visualizer" output="screen" />
</launch>
